### ISSUE TITLE
x-position of sprite is hard to find

### ISSUE THREAD

**Issue by aoloe** [LLK/scratch-gui#4439]
currently, in "feeling" there is a "property of thing" (by default "backgrop# of stage") multiple features item that defaults to the backdrop.

the problem is that one of the most common actions is "x-position of sprite2" and for a kid it's a hard to discover that "x-position" will be part of the list as soon you pick the sprite2 instead of the backdrop.

i see two possible solutions:

- as soon as a second sprite is sprite is created, the right part should automatically switch to the sprite.
- show all possible properties in the left side and gray out the ones that are not available.

**Comment by aoloe** [comment]
to be clear: it's about this block:

![backdrop-of-stage](https://user-images.githubusercontent.com/1857016/51844617-f993a900-2315-11e9-80fe-99671f216236.png)

**Comment by 2br-2b** [comment]
I agree. Maybe there should be 2 separate blocks: one for backdrops, and one for sprites.
